core/src/jvmTest/kotlin/com/ustadmobile/core/controller/XapiPackageContentPresenterTest.kt

```diff
@@ -90,13 +89,12 @@ class XapiPackageContentPresenterTest {
 
         httpd = EmbeddedHTTPD(0, Any(), db, repo)
         httpd!!.start()
-
-        mockXapiPackageContentView = mock(XapiPackageContentView::class.java)
+        mockedView = mock{}
 
         doAnswer { invocation ->
             Thread(invocation.getArgument<Any>(0) as Runnable).start()
             Any()
-        }.`when`<XapiPackageContentView>(mockXapiPackageContentView).runOnUiThread(any())
+        }.`when`<XapiPackageContentView>(mockedView).runOnUiThread(any())
```

Comment:
The best way to do this in kotlin mockito is:
```
whenever(mockedView.runOnUiThread(any()).doAnswer { invocation ->
   Thread(invocation.getArgument<Any>(0) as Runnable).start()
}
```
You can add any if it needs that to finish the closure.